README.md

```diff
@@ -145,41 +145,50 @@ issue. We thank you in advance for your cooperation.
 
 ### Requirements
 
-* C compiler
-* [Go compiler](https://golang.org/dl/)
-* liblzma-dev, libssl-dev and libglib2.0-dev packages
-
-#### LZMA support opt-out
-
-If no LZMA Artifact compression support is desired, you can ignore the `liblzma-dev` package
-dependency and substitute the `make` commands in the instructions below for:
+* C++ compiler
+* cmake
+* libarchive-dev, libboost-all-dev, liblzma-dev and libssl-dev packages
```

Comment:
And lmdb!

Reply:
Fixed. At least according to my test, `liblzma-dev` is not explicitly needed on Ubuntu.